src/Compilers/VisualBasic/Portable/Symbols/Tuples/TupleErrorFieldSymbol.vb

```diff
@@ -94,7 +94,10 @@ Namespace Microsoft.CodeAnalysis.VisualBasic.Symbols
 
         Public Overloads Function Equals(other As TupleErrorFieldSymbol) As Boolean
             Return other Is Me OrElse
-                (other IsNot Nothing AndAlso Me._tupleFieldId = other._tupleFieldId AndAlso Me.ContainingType = other.ContainingType)
+                (other IsNot Nothing AndAlso
+                Me._tupleElementIndex = other._tupleElementIndex AndAlso
+                Me.ContainingType = other.ContainingType AndAlso
+                Me.Name = other.Name)
```

Comment:
Ahould use the same kind of euality as used when creating field symbols. I think it uses string comparison with IdentifierComparer
There are more than one way of doing case-insensitive compares when dealing with unicode. IdentifierComparer encapsulates The "right" way for VB